images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
```

Comment:
Probably not, but it's easier to debug when things go wrong, since you can
shell in.

--brendan

On Thu, Feb 26, 2015 at 8:41 PM, Tim Hockin <notifications@github.com>
wrote:

> In images/master/Dockerfile
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4885#discussion_r25487672>
> :
>
> > @@ -0,0 +1,6 @@
> > +FROM google/debian:wheezy
>
> Do we really need a full debian base? How far are we from being able to
> run FROM SCRATCH? Do we even know what things we would need?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4885/files#r25487672>
> .
>